packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -593,3 +592,9 @@ jobs:
     workflowFile.writeAsStringSync(content);
   }
 }
+
+const permissionsBlock = '''
```

Comment:
Move back to defining inline. Can keep `const` if all workflows need secrets now.